gui/src/components/Taipy/Chart.tsx

```diff
@@ -385,6 +385,13 @@ const Chart = (props: ChartProp) => {
                           ret.z = baseZ;
                       }
                   }
+                  // Hack for treemap charts: create a fallback 'parents' column if needed
+                  // This works ONLY because 'parents' is the third named axis
+                  // (see __CHART_AXIS in gui/utils/chart_config_builder.py)
+                  else if (config.types[idx] === "treemap" && ("values" in ret)) {
```

Comment:
Should you test that parents is indeed the third axis name ?

Reply:
`"values" in ret` => Array.isArray(ret.values) ?

Reply:
> Should you test that parents is indeed the third axis name ?

Well that's what the comment is addressing...